barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_flavor.hpp

```diff
@@ -284,48 +283,13 @@ class UltraFlavor {
 
         std::vector<uint32_t> memory_read_records;
         std::vector<uint32_t> memory_write_records;
-        std::array<Polynomial, 4> sorted_polynomials;
         ProverPolynomials polynomials; // storage for all polynomials evaluated by the prover
 
-        void compute_sorted_accumulator_polynomials(const FF& eta, const FF& eta_two, const FF& eta_three)
-        {
-            // Compute sorted witness-table accumulator
-            compute_sorted_list_accumulator(eta, eta_two, eta_three);
-
-            // Finalize fourth wire polynomial by adding lookup memory records
-            add_plookup_memory_records_to_wire_4(eta, eta_two, eta_three);
-        }
-
-        /**
-         * @brief Construct sorted list accumulator polynomial 's'.
-         *
-         * @details Compute s = s_1 + η*s_2 + η²*s_3 + η³*s_4 (via Horner) where s_i are the
-         * sorted concatenated witness/table polynomials
-         *
-         * @param key proving key
-         * @param sorted_list_polynomials sorted concatenated witness/table polynomials
-         * @param eta random challenge
-         * @return Polynomial
-         */
-        void compute_sorted_list_accumulator(const FF& eta, const FF& eta_two, const FF& eta_three)
-        {
-            auto& sorted_list_accumulator = polynomials.sorted_accum;
-
-            // Construct s via Horner, i.e. s = s_1 + η(s_2 + η(s_3 + η*s_4))
-            for (size_t i = 0; i < this->circuit_size; ++i) {
-                FF T0 = sorted_polynomials[3][i] * eta_three;
-                T0 += sorted_polynomials[2][i] * eta_two;
-                T0 += sorted_polynomials[1][i] * eta;
-                T0 += sorted_polynomials[0][i];
-                sorted_list_accumulator[i] = T0;
-            }
-        }
-
         /**
          * @brief Add plookup memory records to the fourth wire polynomial
          *
-         * @details This operation must be performed after the first three wires have been committed to, hence the
-         * dependence on the `eta` challenge.
+         * @details This operation must be performed after the first three wires have been
```

Comment:
Updated to add_ram_rom_memory_records_to_wire_4. This method has nothing to do with pollokup or lookups, it was just an erroneous function name

Reply:
Ahh I see what happened now. There is an old  comment that points to plookup_auxiliary_widget.hpp but at the time that was implemented, plookup was like a synonym for Ultra because it was the only thing distinguishing Ultra from Turbo. A bit of dusty aztec history..